client/v2/shield/init.go

```diff
@@ -1,10 +1,32 @@
 package shield
 
-func (c *Client) Initialize(master string) error {
+import "bytes"
+
+func (c *Client) Initialize(master string) (string, error) {
 	in := struct {
 		Master string `json:"master"`
 	}{
 		Master: master,
 	}
-	return c.post("/v2/init", in, nil)
+
+	out := struct {
+		DisasterKey string `json:"disaster_key"`
+	}{
+		DisasterKey: "",
+	}
+
+	err := c.post("/v2/init", in, &out)
+
+	return out.DisasterKey, err
+}
+
+func (c *Client) SplitKey(s string, n int) string {
+	var buffer bytes.Buffer
+	for i, rune := range s {
+		buffer.WriteRune(rune)
+		if i%n == (n-1) && i != (len(s)-1) {
+			buffer.WriteRune('\n')
+		}
+	}
+	return buffer.String()
```

Comment:
There's a more legible way of doing this (breaking a long string into lines) using string slices; we should investigate.